packages/salesforcedx-vscode-core/package.json

```diff
@@ -823,6 +823,11 @@
           "type": "boolean",
           "default": true,
           "description": "%experimental_templates_library_description%"
+        },
+        "salesforcedx-vscode-core.functions_pull_dependencies": {
```

Comment:
We should change the use of `pull` for `install` since that's the term used by NodeJS

Reply:
@lcampos - do you think we should use "install" instead of "pull" in the error msg text?

Reply:
Yes, `pull` is a git term and NodeJS uses `install`. Everything related to NodeJS Functions should use `install` if we have it using `pull`